### ISSUE TITLE
Unsupervised VOS Evaluation

### ISSUE THREAD

**Issue by Spritea** [henghuiding/MOSE-api#14]
Hi, thanks for the great dataset!

I am interested in the unsupervised VOS part. Although the meatafiles for training set and validation set have included the `first_frame_exhaustive_anno` part to denote whether the first frame is exhaustively annotated, the evaluation server on CodaLab seems to not include the specific results for the unsupervised VOS setting.

If that's the case, do we have any other way to evaluate the unsupervised VOS setting and so we can compare it with the Table 5 results in the MOSE paper? Thank you!

**Comment by henghuiding** [comment]
Hi,

The unsupervised VOS evaluation is under construction and will reopen when ready. You can send the .png files to us and we return the J&F results. BTW, we will host a [challenge](https://henghuiding.github.io/MOSE/ChallengeCVPR2024) on CVPR'24. Thank you.

**Comment by Spritea** [comment]
Thank you for the prompt explanation! Will ask for your kind help when we get the .png files.
